style.css

```diff
@@ -316,11 +316,6 @@ section.container {
   margin: 300px 690px 300px 690px;
 }
 
-.form {
-  width: 375px;
-  height: 779px;
-}
-
 .form-section {
   background-image: url("/images/header-shapes\ mobile.svg");
   background-repeat: no-repeat;
```

Comment:
- [ ] As requested by the previous reviewer kindly have this section laid out using flexbox since this is a major requirement of the project.